src/apps/schema/src/appRevamp/components/ModelsTable/index.tsx

```diff
@@ -159,7 +161,18 @@ export const ModelsTable = ({ search }: Props) => {
             cursor: "pointer",
           },
         }}
+        components={{
+          NoRowsOverlay: () => <Box></Box>,
```

Comment:
Yes because there's a default empty table message on the datagrid. But I think I already fixed this one by adjusting the height of the table when there are no items. So I'll just remove this one on my next commit.